starboard/CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ since the version previous to it.
 
 ## Version 16
 
+### Added standard POSIX socket getaddrinfo/freeaddrinfo APIs.
```

Comment:
This is not `stat` related, why is it part of this PR?

Reply:
If you look at the file in the PR in total (not just the later commits) it shows the correct history -- only the stat changelog message is added.